recipes/ot-commissioner/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "20200404":
+    sha256: 04a8cd1b3e79ef8f42341d50c32916806e1313bbcc76292f7f57c36ade371b18
+    url: https://github.com/gocarlos/ot-commissioner/archive/643e7e8cf803f3fd8daa09673d49badc43b4b385.zip
```

Comment:
Why do you use your fork?

Let's use the upstream and the tagged version.
https://github.com/openthread/ot-commissioner/releases/tag/v0.0.1